tools/IoTEdgeDevOps/IoTEdgeDashboard/appsettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "AppConfig": {
+    "PersonalAccessToken": "g5zysckiu4u7kskuykdnbwzymdo6em2g2tim3fxqug7pufz42laa"
```

Comment:
Should the `g5zysckiu4u7kskuykdnbwzymdo6em2g2tim3fxqug7pufz42laa` be stored in Key Vault?

Reply:
Good catch!  I accidentally checked in this token, it is already expired.